r/R/dplyr-eval.R

```diff
@@ -201,7 +201,12 @@ try_arrow_dplyr <- function(expr) {
   parent <- caller_env()
   # Make sure that the call is available in the parent environment
   # so that we can use it in abandon_ship, if needed
-  evalq(call <- match.call(), parent)
+  # (but don't error if we're in some weird context where we can't get the call,
+  # which could happen if you're code-generating or something?)
```

Comment:
Would it be worth specifically mentioning tidyquery here? I guess we have the PR links so that we can come back to it.

Reply:
🤷  up to you, feel free to ```suggestion some words

Reply:
I think I convinced myself the PR is documentation enough while writing that comment 😂 Describing code-gen should be a good flag too anyway